README.md

```diff
@@ -1 +1,25 @@
-# rules-technical-challenge-el
\ No newline at end of file
+# RoboCup Brasil Small Size League Entry Level Technical Challenges
+
+This is the AsciiDoc source for the official RoboCup Small Size League Entry Level technical challenges rules.
+
+Each year has its own directory with its .adoc files containing rules for each technical challenge that will or had happened in that specific year.
```

Comment:
I think we can provide a brief description of the technical challenge and the impact that challenges have on the competition. @PedroPizarro

Reply:
@ersaraujo Done